docs/home/contribute/style-guide.md

```diff
@@ -172,6 +172,17 @@ apiVersion: v1 # earlier versions use...
 kind: Pod
 ...
 ```
+## Accessibility Guidelines
+By law in many countries, content must be accessible to people who have disabilities, such as visual impairments. Anything that the reader must know or do needs to be in text, too, not just in images.
+
+<table>
+  <tr><th>Content</th><th>Usage</th></tr>
+  <tr><td>Illustrations</td><td> Include figure tags &lt;figure&gt; and alternative text tags &lt;alt&gt;. Introduce illustrations or code listings with a clear lead-in sentence. Provide detailed instructions and explanations after the image or code example.</td></tr>
+  <tr><td>Code</td><td>Do not use screen captures or other graphics to show code, commands, or screen output that readers must do something with or know about. Ensure that all code is in code font so that it is readable by software for the vision-impaired.</td></tr>
+  <tr><td>Tables</td><td>Avoid rowspan or colspan (span tags) if possible. Use simple tables.</td></tr>
+    <tr><td>Date</td><td>Spell out months, as in 20 November 2018, rather than 20 Nov 2018.</td></tr>
+    <tr><td>Text</td><td>Do not use red or green text or highlighting anywhere, because colorblind people cannot distinguish it from other text or elements.</td></tr>
```

Comment:
When I first read this, I interpretted it as "don't use syntax highlighting at all", when I think what you're likely intending is "don't rely on syntax highlighting to differentiate the content when that's needed to your explanatory content". Could you clarify that or consider how to reword this? I do think there's value in things like syntax highlighting, even if it's not ultimately consumable by some portion of our readers.

Reply:
You are right @heckj . We need to look into: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html and reword the guidelines accordingly.